packages/x-codemod/package.json

```diff
@@ -49,6 +49,6 @@
     "directory": "build"
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=20.19"
```

Comment:
To determine first if it's a breaking change. If the `yargs` update was from before v8, we could probably let this pass. If it's from later, then technically the breaking change was in that dependency update.

Reply:
That's unfortunate. Technically it's a breaking change and needs to be reverted. Pragmatically, for this specific package, I personally wouldn't have a problem of letting it slide. Up to you